### DISCUSSION TITLE
Functions that work for Pro workspaces

### DISCUSSION THREAD

**Discussion by frithjofv** [microsoft/semantic-link-labs#311] [General]
Hi,

Is there an easy way to find out which Semantic Link and Semantic Link Labs functions that work for items in Pro workspaces?

Thanks for all the awesome features you're bringing about, by the way! Cheers

**Comment by m-kovalsky** [comment]
I don’t really have such a list. I believe anything which requires XMLA R/W will require premium or fabric. I imagine that functions which are just api wrapper calls should work fine.

**Reply by frithjofv** [reply]
Thanks for your reply @m-kovalsky 

Cheers
